.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: test
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['3.1', '3.0', '2.7']
```

I suppose you meant 3.2 instead of 2.7?

```suggestion
        ruby-version: ['3.2', '3.1', '3.0']
```